docs/readthedocs/source/doc/Nano/Overview/pytorch_cuda_patch.md

```diff
@@ -0,0 +1,56 @@
+# PyTorch CUDA Patch
+
+BigDL-Nano also provides CUDA patch to help you run CUDA code without GPU. This patch will replace CUDA operations with equivalent CPU operations, so after applying it, you can run CUDA code on your CPU without changing any code.
+
+This patch is included in `bigdl.nano.pytorch.patch_torch`, you can call `patch_torch(cuda_to_cpu=True)` to enable it (the `cuda_to_cpu` parameter defaults to `True`). You can also use it alone by `bigdl.nano.pytorch.patching.gpu_cpu.patch_cuda`.
```

Comment:
we could just introduce `from bigdl.nano.pytorch.patching import patch_cuda` to our users.

Reply:
but `unpatch_cuda` is not in `bigdl.nano.pytorch.patching`

Reply:
you are right, we should make it available in `bigdl.nano.pytorch.patching` and only introduce `bigdl.nano.pytorch.patching` to users. `patch_torch` could be deprecated later.

Reply:
changed it